lib/resqutils/spec/resque_helpers.rb

```diff
@@ -26,7 +26,11 @@ def process_resque_job(expected_class, queue=nil, &block)
           end
     raise "No jobs on #{queue}" if job.nil?
     klass = job["class"].constantize
-    klass.should == expected_class
+    if Gem.loaded_specs['rspec'].version >= Gem::Version.new('3')
```

Comment:
Ah, nice!  I tried another way to figure this out before and couldn't.  This looks good!